src/components/BadgeSelect.tsx

```diff
@@ -0,0 +1,98 @@
+import has = require('lodash/has');
+import includes = require('lodash/includes');
+import * as React from 'react';
+import { BadgeSelectProps } from 'rendition';
+import styled from 'styled-components';
+import Badge from './Badge';
+import DropDownButton from './DropDownButton';
+import Text from './Text';
+
+interface BadgeSelectState {
```

Comment:
NIT: how about moving this right above the component?
I was getting lost toooooo easily before starting following that rule.

Reply:
can do